test_plans/guppy/test_plan.md

```diff
@@ -0,0 +1,100 @@
+		And match the contents of test_response_1.json
+
+Feature: A user can obtain the total count of the query result with a total count aggregation.
+	Background: (#1)
+
+	Scenario: Mingfei wants a total count of patients matching the filter in the scenario labeled (*).
+		When Mingfei makes an API request to /guppy/graphql with test_query_2.json
+		Then the response will be successful with status code 200
+		And match the contents of test_response_2.json
+
+Feature: A user can obtain overview of data using aggregation query
+	Background: (#2)
+
+	Scenario: Mingfei wants to render a set of visualizations summarizing data in the commons.
+		When Mingfei makes an API request to /guppy/graphql with test_query_6.json
+		Then the response will be successful with status code 200
+		And match the contents of test_response_6.json
+
+Feature: A user can analyze buckets of numeric data split up with the rangeStep and binCount parameters.
+	Background: (#2)
+
+	Scenario: Mingfei wants to make multiple histograms describing the BMI parameter to gain an understanding of its distribution.
+		When Mingfei makes an API request to /guppy/graphql with test_query_7.json
+		Then the response will be successful with status code 200
+		And match the contents of test_response_7.json
+
+Feature: A user can obtain bin counts of data classifications using text aggregation.
+	Background: (#1)
+
+	Scenario: Mingfei wants a high-level overview of the data in the database as it pertains to stroke occurence and age groups represented.
+		When Mingfei makes an API request to /guppy/graphql with test_query_3.json
+		Then the response will be successful with status code 200
+		And match the contents of test_response_3.json
+
+Feature: A user can compute statistical summaries of a numerical field with parameters like min, max, avg, sum, count.
+	Background: (#1)
+
+	Scenario: Mingfei wants a high-level overview of the data in the database as it pertains to stroke occurence and age groups represented.
+		When Mingfei makes an API request to /guppy/graphql with test_query_4.json
+		Then the response will be successful with status code 200
+		And match the contents of test_response_4.json
+
+Feature: A user can list the fields on a document type with a mapping query.
+	Background: (#1)
+
+	Scenario: Mingfei would like to list the fields on the case document.
+		When Mingfei makes an API request to /guppy/graphql with test_query_5.json
+		Then the response will be successful with status code 200
+		And match the contents of test_response_5.json
+
+Feature: A user can make requests exceeding 10k records to /download endpoint.
+	Background: (#2)
+
+	Scenario: Mingfei wants to execute the Scenario labeled (*) without worrying about paginating the results.
+		When Mingfei makes an API request to /guppy/download with test_query_8.json
+		Then the response will be successful with status code 200
+		And match the contents of test_response_8.json
+
```

Comment:
@ZakirG I am not sure if you have merged test plan and gherkin feature file into one? I am not sure whether to review it as a test plan or as a gherkin file. The gherkin feature file has the extension '.feature' and we want to have both test plan as well as gherkin file for guppy (but separated).

Reply:
Oh I see ok. So I should make a separate test plan file in this format? https://github.com/uc-cdis/gen3-qa/blob/master/test_plans/pidgin/test_plan.md